yb-voyager/cmd/cutoverInitiate.go

```diff
@@ -24,7 +24,7 @@ import (
 var prepareForFallBack utils.BoolStr
 
 var cutoverInitiateCmd = &cobra.Command{
-	Use:   "initiate",
+	Use:   "target",
 	Short: "Initiate cutover to YugabyteDB",
 	Long:  `Initiate cutover to YugabyteDB`,
```

Comment:
Can you elaborate more in `Long` help text? e.g.
* after the cutover user should run `import schema --post-import-data`
* what command can be used to get the current status of the cutover.
*